nomad/jitsi-meet-web.hcl

```diff
@@ -742,23 +742,6 @@ config.maxFullResolutionParticipants = 1;
 {{ end -}}
 
 if (!config.hasOwnProperty('videoQuality')) config.videoQuality = {};
```

Comment:
we can remove this bit too, since we're not setting any other videoQuality configs here.

Reply:
I want to leave this in since I will soon be setting preferred codecs here once the AV1 patch in the bridge lands